wordpress-mobile/gutenberg-mobile
# iOS DarkMode toolbar buttons

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/17356
## Description


This PR allow us to add different styles to DashIcons inside IconButton component depending on the the style theme (Dark/Light mode).

It also enhance the DarkMode theme for Toolbar buttons (and the toolbar separator)

With this change, is not needed to send `ariaPressed` prop from `IconButton`. This is a change that [was requested](https://github.com/WordPress/gutenberg/pull/13977#issuecomment-468731745) long ago, (even though we still are sending it button states).

A second idea is to send styles directly to DashIcon from the Button, to avoid having to declare styles for every new class. I'm not sure if this is something we want to change though.

As always any moment is welcome! 

![toolbar](https://user-images.githubusercontent.com/9772967/64407508-bf953d00-d084-11e9-9b51-22f68544e3dd.png)

To test:
- Run the example app with Xcode 11 beta
- Check that the toolbar colors are correct on the toolbar (following the screenshots)
- Check that the rest of the icons in the app shows properly
  - In particular the BottomSheet icons (Add link / Image settings)
- Check that the icon Subscript is colored properly (Numbers on H1, H2, etc...)

cc @iamthomasbishop
